app/controllers/tagging_spreadsheets_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
     if tagging_spreadsheet.valid?
       tagging_spreadsheet.save!
       InitialTaggingImport.perform_async(tagging_spreadsheet.id)
-      redirect_to tagging_spreadsheets_path
+      redirect_to tagging_spreadsheet, success: "Spreadsheet uploaded. It will be imported within a few minutes."
```

Comment:
Sorry, David, I wrote that message from my phone and the autocorrect didn't really help much :)